native/src/modules/common/components/TimeStamp.js

```diff
@@ -29,18 +29,19 @@ type PropsType = {|
   formatter: DateFormatter,
   t: TFunction,
   language: string,
-  theme: ThemeType
+  theme: ThemeType,
+  useText?: boolean
```

Comment:
I think `showText` or similar would be better. This looks like a hook :thinking: